src/test/regress/expected/non_super_user_object_metadata.out

```diff
@@ -0,0 +1,303 @@
+SELECT create_distributed_function('test_function(int)');
+ERROR:  must be owner of function test_function
+RESET ROLE;
+SET search_path TO local_schema;
+ALTER SEQUENCE test_sequence OWNER TO test_user;
+ALTER FUNCTION test_function(int) OWNER TO test_user;
+SELECT start_metadata_sync_to_node('localhost', :worker_1_port);
+ start_metadata_sync_to_node 
+-----------------------------
+ 
+(1 row)
+
+SET ROLE test_user;
+SET search_path TO local_schema;
+-- Show that we can distribute table and function after
+-- having required ownerships
+SELECT create_distributed_table('dist_table', 'a');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+SELECT create_distributed_function('test_function(int)');
+ create_distributed_function 
+-----------------------------
+ 
+(1 row)
+
+-- show that schema, types, function and sequence has marked as distributed
+-- on the coordinator node
+RESET ROLE;
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema'::regnamespace::oid;
+ pg_identify_object_as_address 
+-------------------------------
+ (schema,{local_schema},{})
+(1 row)
+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema.mood'::regtype::oid;
+ pg_identify_object_as_address 
+-------------------------------
+ (type,{local_schema.mood},{})
+(1 row)
+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema.test_type'::regtype::oid;
+   pg_identify_object_as_address    
+------------------------------------
+ (type,{local_schema.test_type},{})
+(1 row)
+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema.test_sequence'::regclass::oid;
+        pg_identify_object_as_address         
+----------------------------------------------
+ (sequence,"{local_schema,test_sequence}",{})
+(1 row)
+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema.dist_table_e_seq'::regclass::oid;
+          pg_identify_object_as_address          
+-------------------------------------------------
+ (sequence,"{local_schema,dist_table_e_seq}",{})
+(1 row)
+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema.test_function'::regproc::oid;
+            pg_identify_object_as_address            
+-----------------------------------------------------
+ (function,"{local_schema,test_function}",{integer})
+(1 row)
+
+-- show those objects marked as distributed on metadata worker node as well
+SELECT run_command_on_workers($$SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object where objid = 'local_schema'::regnamespace::oid;$$);
+              run_command_on_workers              
+--------------------------------------------------
```

Comment:
needs ORDER BY so that worker order does not change in the output. Probably applies to all.